mix.exs

```diff
@@ -1,38 +1,35 @@
 defmodule Rope.Mixfile do
   use Mix.Project
 
-  def version do
-    "v0.1.1"
-  end
-
-  def source_url do
-    "https://github.com/copenhas/ropex"
-  end
-
   def project do
-    [ app: :rope,
-      version: version,
-      elixir: "~> 0.10.0",
+    [
+      app: :rope,
+      version: "0.1.2",
+      elixir: "~> 1.5",
+      start_permanent: Mix.env == :prod,
+      deps: deps(),
+
+      # Docs
       name: "ropex",
-      source_url: source_url,
-      deps: deps,
-      docs: [
-        source_url_pattern: "#{source_url}/blob/#{version}/%{path}#L%{line}"
-      ]
+      source_url: "https://github.com/ijcd/ropex",
```

Comment:
Since this is being contributed back to my fork, should this be `copenhas/ropex`?